### PR TITLE
820 fix links

### PR DESCRIPTION
**Pull request by rossfarrugia**
@thomas-neitmann i dont think i need to run devtools::documents() as i only updated R/data.R but let me know if i'm mistaken.

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://roche-gsk.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://roche-gsk.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x ] Update to all relevant roxygen headers and examples 
- [x ] Run `devtools::documents()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x ] Address any updates needed for vignettes and/or templates
- [x ] Update `NEWS.md`
- [x ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://roche-gsk.github.io/admiral/reference/index.html)" page. 
- [x ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x ] Link the issue so that it closes after successful merging. 
- [x ] Address all merge conflicts and resolve appropriately 
- [x ] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
This PR includes site link fixes since moving to pharmaverse website and also in the README now included ref="main" as per discussion on developer Slack since we renamed master branch.

**Comment by rossfarrugia** [comment]
@thomas-neitmann as i was updating README i took the opportunity to add a pharmaverse badge as per our draft site page: https://pharmaverse.org/contribute/badges/. over time we're hoping this will be like a mark of honour for recommended packages

**Comment by mstackhouse** [comment]
@rossfarrugia I do think you want to run `devtools::document()`. The data.R file generates these ,rd files 
- adae.Rd
- adcm.Rd
- adex.Rd
- adsl.Rd
- advs.Rd
- ex_single.Rd

Each of those contains the only roche-gsk link.

**Comment by rossfarrugia** [comment]
thanks @mstackhouse think i've got that working now but @thomas-neitmann can double check next week. i had issues with devtools::documents() saying _Error: 'documents' is not an exported object from 'namespace:devtools'_. but devtools::check() worked and the .Rd files got updated fine. i think its that i need to update my RStudio project to link the new repo. shout if anyone's seen this before and has any other advice!

**Comment by mstackhouse** [comment]
@rossfarrugia  no "s" :)

https://devtools.r-lib.org/reference/document.html

File: NEWS.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ are not recorded in the data. (#589)
 
 - New vignette [Development Process](../articles/development_process.html) and improvements made to contribution vignettes (#765 & #758)
 
+- GitHub repo moved to pharmaverse org and associated broken site links fixed
```
Comment (thomas-neitmann) [inline NEWS.md:18]: Please include the issue number. I would guess there was one for the move and another one to fix the links.

File: README.md [hunk 2]
```diff
@@ -20,16 +21,16 @@ In the meantime you can install the latest release of the package directly from
 if (!requireNamespace("remotes", quietly = TRUE)) {
   install.packages("remotes")
 }
-remotes::install_github("Roche-GSK/admiral.test") # This is a required dependency of {admiral}
-remotes::install_github("Roche-GSK/admiral")
+remotes::install_github("pharmaverse/admiral.test") # This is a required dependency of {admiral}
```
Comment (thomas-neitmann) [inline README.md:24]: I will change the default branch of {admiral.test} to `main` as well in order to be consistent. Thus, please add `ref = "main"` here as well.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

**Comment by rossfarrugia** [comment]
thanks @mstackhouse - thats what happens when i work late on a friday... ;)
will make those updates now @thomas-neitmann

**Comment by rossfarrugia** [comment]
@thomas-neitmann this one should be good to go now. had to resolve some merge conflicts for README and NEWS. this vignette really helped https://pharmaverse.github.io/admiral/articles/git_usage.html

**Comment by rossfarrugia** [comment]
because this one touches so many programs in so many places, every time we make a new merge to develop we're likely to get more merge conflicts until this one itself merges :)

**Comment by rossfarrugia** [comment]
@bms63 @thomas-neitmann can whoever has earliest availability please review and merge this one? i saw the "roche-gsk" comment come up again from Ben's review of #875 - we're always going to get these and merge conflicts until we merge this PR that affects so many programs.

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**